frontend/app/src/components/accounts/blockchain/XpubInput.vue

```diff
@@ -115,11 +104,30 @@ const keyTypeListData = computed<XpubType[]>(() => {
   );
 });
 
-const rules = [
-  (v: string) => {
-    return !!v || t('account_form.validation.xpub_non_empty').toString();
+const rules = {
+  xpub: {
+    required: helpers.withMessage(
+      tc('account_form.validation.xpub_non_empty'),
+      required
+    )
+  },
+  derivationPath: {
+    basic: (v: string | null) => !v || v
```

Comment:
hmm isn't it always `true` then?

Reply:
yes we can also return always true here but we still have to have a rule otherwise the external error messages don't work